src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -149,6 +149,11 @@ const AppWrapper = (props: { children: ReactNode }) => {
   }, []);
 
   useEffect(() => {
+    if (authentication.loggedIn) {
```

Comment:
If we are going to update `setPauseQueueIncomingRequest` every time `authentication.loggedIn` changes, then I think we can remove the `state.queueIncomingRequest` updates in the `login` and `logout` methods of `stateCache.ts` - and just update it here